### PR TITLE
chore: remove claim outputs frunction

### PR DESCRIPTION
**Pull request by hauju**
# Pull Request Template

## Motivation and Context

Explain why this change is necessary or the issue it addresses.

## Summary

Provide a short summary of changes in this pull request.

- Change 1
- Change 2

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [ ] Code follows project style and guidelines.
- [ ] No redundant or duplicate code.
- [ ] No added new dependencies without clear justification and approval.

### Documentation

- [ ] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] Added a CHANGELOG entry for major changes.

### Testing

- [ ] Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.
- [ ] Tests pass locally.

### Build & CI/CD

- [ ] Confirmed no CI/CD pipeline issues.
- [ ] Updated build scripts where necessary.

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

File: sdk/docs/SDK Modules/KYC Onboarding.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+
+``` mermaid
+graph LR
+  A[Start] --> B{Error?};
+  B -->|Yes| C[Hmm...];
+  C --> D[Debug];
+  D --> B;
+  B ---->|No| E[Yay!];
+```
```
Comment (antbern) [inline sdk/docs/SDK Modules/KYC Onboarding.md:10]: Should this be here?

**Review by antbern — CHANGES_REQUESTED** [review]
Nice! I think there is another `run_all_examples.sh` for `wasm` bindings that might also need to be edited 💯

File: sdk/docs/SDK Modules/KYC Onboarding.md (hunk 1 above)
Comment (hauju) [reply sdk/docs/SDK Modules/KYC Onboarding.md]: No, but diagrams looks nice in mkdocs :D

**Review by antbern — APPROVED** [review]

**Review by kristifidani — APPROVED** [review]
